Address usability experience for `azsdk verify-setup` (#12976)
Azure/azure-sdk-tools: eng/common/instructions/azsdk-tools/verify-setup.instructions.md

## Output
Display results in a user-friendly and concise format, highlighting any missing dependencies that need to be addressed and how to resolve them.

> When Python tool requirements fail (azpysdk, sdk_generator, ghtools, pytest), provide additional context about the `AZSDKTOOLS_PYTHON_VENV_PATH` environment variable if the user seems to be having setup issues. The verify-setup tool can only check Python requirements within the virtual environment specified by this environment variable.

Reviewer: We should probably avoid hard-coding the requirements in the prompt in case they change. Everything else lgtm!

```suggestion
When Python tool requirements fail, inform the user about the `AZSDKTOOLS_PYTHON_VENV_PATH` environment variable if they have setup issues. The verify-setup tool can only check Python requirements within the virtual environment specified by this environment variable.
```
